### PR TITLE
Host troubleshooting pages on fastly.help for common CLI issues

### PR DESCRIPTION
**Pull request by kailan**
The first page will be live at https://fastly.help/cli/ecp-feature after this PR and https://github.com/fastly/Developer-Portal/pull/653 have been merged.

### CODE REVIEW

**Review by Integralist — APPROVED** [review]
LGTM. 

But just to be explicit: if the timeline for merging the other PR is expected to take a while, let's **_not_** merge this PR until the other PR is merged/deployed. Otherwise it will make cutting a new release in the mean time trickier (because to cut a new release with these changes will result in a non-working remediation URL being exposed).

**Comment by kailan** [comment]
Agreed! I will wait until the other PR is merged before merging this one.

File: pkg/compute/deploy.go
Comment (triblondon) [inline pkg/compute/deploy.go:427]: I think we concluded that most terminals will recognise this as a URL without the 'https://', so consider removing it?
```suggestion
				Remediation: "For more help with this error see fastly.help/cli/ecp-feature",
```

**Review by triblondon — APPROVED** [review]

**Pull request merged**
